### PR TITLE
C343214 Filter users by tags (volaris) (TaaS)

### PR DESCRIPTION
**Pull request by ylatysh**
[C373214](https://foliotest.testrail.io/index.php?/cases/view/343214)
[Allure report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/6470/allure/)

image

### CODE REVIEW

File: cypress/e2e/users/filter-users-by-tags.cy.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  });
+
+  after('Delete a user', () => {
+    cy.getAdminToken();
+    Users.deleteViaApi(userData.userId);
+    Users.deleteViaApi(existingUser.userId);
+  });
+
+  it('C343214 Filter users by tags (volaris) (TaaS)', { tags: ['extendedPath', 'volaris'] }, () => {
+    // 1
```
Comment (manowarqqq) [inline cypress/e2e/users/filter-users-by-tags.cy.js:39]: Either remove numbers or add more context

File: cypress/e2e/users/filter-users-by-tags.cy.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    });
+  });
+
+  after('Delete a user', () => {
+    cy.getAdminToken();
+    Users.deleteViaApi(userData.userId);
+    Users.deleteViaApi(existingUser.userId);
+  });
+
+  it('C343214 Filter users by tags (volaris) (TaaS)', { tags: ['extendedPath', 'volaris'] }, () => {
```
Comment (manowarqqq) [inline cypress/e2e/users/filter-users-by-tags.cy.js:38]: criticalPath

File: cypress/support/fragments/users/usersSearchPane.js [hunk 3]
```diff
@@ -70,4 +75,17 @@ export default {
     cy.do(Button('Actions').click());
     cy.expect(Button('Lost items requiring actual cost').absent());
   },
+
+  verifyTagOption(tagName) {
```
Comment (manowarqqq) [inline cypress/support/fragments/users/usersSearchPane.js:79]: verify method shouldn't do action. btw method is not used

File: cypress/e2e/users/filter-users-by-tags.cy.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  });
+
+  after('Delete a user', () => {
+    cy.getAdminToken();
+    Users.deleteViaApi(userData.userId);
+    Users.deleteViaApi(existingUser.userId);
+  });
+
+  it('C343214 Filter users by tags (volaris) (TaaS)', { tags: ['criticalPath', 'volaris'] }, () => {
+    cy.login(userData.username, userData.password);
```
Comment (re-jas) [inline cypress/e2e/users/filter-users-by-tags.cy.js:39]: please login in before hook using. 
`cy.loginAsAdmin(userData.username, userData.password, { path: TopMenu.usersPath, waiter: UsersSearchPane.waitLoading });`

**Review by re-jas — APPROVED** [review]

**Pull request merged**